### PR TITLE
Security flag fixes

### PR DESCRIPTION
**Pull request by jandryuk**
Fixes for -Werror=format-security and -Werror=unused-result

### CODE REVIEW

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/#/builders/1/builds/7047

File: input.c [hunk 1]
```diff
@@ -1852,7 +1852,7 @@ static void input_led(int onoff, int led)
             ev.type = EV_LED;
             ev.code = led;
             ev.value = onoff;
-            write(input_dev.fds[i], &ev, sizeof(struct input_event));
+            (void)!write(input_dev.fds[i], &ev, sizeof(struct input_event));
```
Comment (jean-edouard) [inline input.c]: Why the `!` here? Is it necessary to silence the warning?

Comment (jandryuk) [reply input.c]: Yes.  The commit message includes "A void cast is insufficient to silence the error, but the additional negation does
the trick."  I can update if you want something more.

Comment (jean-edouard) [reply input.c]: Sorry about that, missed the commit message...

**Pull request merged**